From qiyong@fc-cn.com  Mon Aug  6 20:07:56 2007
From: Qi Yong <qiyong@fc-cn.com>
Date: Tue, 12 Jun 2007 13:06:49 +0800
Subject: HOWTO: latest lxr url address changed
To: gregkh@suse.de
Cc: qiyong@virgo.fc-cn.com
Message-ID: <20070612050648.GA27739@localhost.localdomain>
Content-Disposition: inline


Hello,

I've noticed that in Document/HOWTO the url address:
http://sosdg.org/~coywolf/lxr/
has changed to
http://users.sosdg.org/~qiyong/lxr/
from the website.

	-- qiyong

Signed-off-by: Qi Yong <qiyong@fc-cn.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 Documentation/HOWTO             |    4 ++--
 Documentation/SubmittingPatches |    2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

--- a/Documentation/HOWTO
+++ b/Documentation/HOWTO
@@ -208,7 +208,7 @@ tools.  One such tool that is particular
 Cross-Reference project, which is able to present source code in a
 self-referential, indexed webpage format. An excellent up-to-date
 repository of the kernel code may be found at:
-	http://sosdg.org/~coywolf/lxr/
+	http://users.sosdg.org/~qiyong/lxr/
 
 
 The development process
@@ -384,7 +384,7 @@ One of the best ways to put into practic
 bugs reported by other people. Not only you will help to make the kernel
 more stable, you'll learn to fix real world problems and you will improve
 your skills, and other developers will be aware of your presence. Fixing
-bugs is one of the best ways to earn merit amongst the developers, because
+bugs is one of the best ways to get merits among other developers, because
 not many people like wasting time fixing other people's bugs.
 
 To work in the already reported bug reports, go to http://bugzilla.kernel.org.
--- a/Documentation/SubmittingPatches
+++ b/Documentation/SubmittingPatches
@@ -560,7 +560,7 @@ NO!!!! No more huge patch bombs to linux
   <http://marc.theaimsgroup.com/?l=linux-kernel&m=112112749912944&w=2>
 
 Kernel Documentation/CodingStyle:
-  <http://sosdg.org/~coywolf/lxr/source/Documentation/CodingStyle>
+  <http://users.sosdg.org/~qiyong/lxr/source/Documentation/CodingStyle>
 
 Linus Torvalds's mail on the canonical patch format:
   <http://lkml.org/lkml/2005/4/7/183>
